plugins/test/unit/plugins/migrations/test_0042_add_suse_fields_to_errata.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from pulp.server.db.migrate.models import _import_all_the_way
+import mock
+
+migration = _import_all_the_way('pulp_rpm.plugins.migrations.0042_add_suse_'
+                                'fields_to_errata')
+
+
+class TestMigrate(unittest.TestCase):
+    """
+    Test the migrate() function.
+    """
+    def setUp(self):
+        super(TestMigrate, self).setUp()
+        self.fake_errata = {
+            '_id': 'abc123'
+        }
+
+    @mock.patch.object(migration, 'get_collection')
+    def test_calls_correct_functions(self, mock_get_collection):
```

Comment:
Can you also add tests for re-run cases?
Both fields are already there, so no db `update` calls
One field is there and it's not changed after migration, just the missing one is added.

Reply:
@goosemania I think it's hard making a unit test for "One field is there and it's not changed after migration, just the missing one is added." Since I'm setting `mock_get_collection.return_value.find.return_value.batch_size.return_value` to test. Do you have any suggestions for this?